package.json

```
@@ -36,9 +36,9 @@
   "peerDependencies": {
     "redux": ">=3 <5",
     "redux-thunk": "2.x.x",
-    "immer": ">=1 <7",
+    "immer": ">=1 <=8.x.x",
```

Can we stick with the style used for the other deps?
```suggestion
    "immer": ">=1 <9",
```